strfuscator/Cargo.toml

```diff
@@ -1,13 +1,14 @@
 [package]
 name = "strfuscator"
 version = "1.0.0"
-edition = "2018"
+edition = "2021"
 authors = [ "bezrazli4n0" ]
 license = "MIT"
 repository = "https://github.com/bezrazli4n0/strfuscator"
 description = "Compile-time easy to use string encoder"
 keywords = [ "encode", "string", "compile", "decode", "encrypt", "xor" ]
 readme = "README.md"
+resolver = "2"
```

Comment:
See prev comment.

Reply:
This needed here, because workspaces by default have [resolver=1](https://doc.rust-lang.org/edition-guide/rust-2021/default-cargo-resolver.html#details).